### ISSUE TITLE
[Feature Request] Windows autostart & minimized (tray icon)

### ISSUE THREAD

**Issue by elhertz** [bladeSk/StayFocused#10]
It'd be nice to have an option for it to autostart when you boot up windows, also minimized.

**Comment by bladeSk** [comment]
Yep, that's pretty much a must for daily usage. Adding a switch to start minimized should be easy enough and I'll also look into creating an autostart shortcut.

**Comment by bladeSk** [comment]
Closing in favor of https://github.com/bladeSk/StayFocused/issues/10

**Comment by jenda69** [comment]
It's been four years and no update. So I created an autohotkey script in the Stay Focused directory and put its shortcut inside the startup directory (Win+R -> shell:startup)

```AHK
Run, StayFocused.exe
WinWait, Stay Focused
WinHide, Stay Focused
 ````

This one is even simpler and also works, but if you click Stay Focused icon in tray, you must do it twice to open the window for the first time.
```ahk
Run, StayFocused.exe, , Hide
```
